src/piyushPJun20/RemoveArrayList.java

```diff
@@ -0,0 +1,26 @@
+package piyushPJun20;
+
+import java.util.ArrayList;
+
+public class RemoveArrayList {
+	void remove(ArrayList<String> name) {
+		for (int index = 0; index < name.size(); index++) {
+			if (name.get(index) == ("Techno")) {
+				name.remove(index);
+			}
+		}
+		System.out.println(name);
```

Comment:
Looks good.  As an alternative , you can just write one line in the for loop " name.remove(name.indexOf("Techno")) " , if you want to avoid the if statement.